CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## 0.15.0-rc.1 / 2018-03-22
+
+* [CHANGE] The default group_by is no labels. (#1287)
+* [CHANGE] Set global SMTP hello to "localhost" by default (#1290)
```

Comment:
This was already this way down in Go's libraries, this commit was only for the sake of clarity and should be omitted from the changelog.